turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -1503,24 +1510,25 @@ impl FileSystemPath {
     /// depend on the order.
     #[turbo_tasks::function]
     pub async fn read_dir(self: Vc<Self>) -> Result<Vc<DirectoryContent>> {
-        match &*self.await?.fs.raw_read_dir(self).await? {
+        let fs = self.await?.fs;
```

Comment:
```suggestion
        let this = self.await?
        let fs = this.fs;
```

Usually we use this pattern to cache `self.await?`. It's also used a few lines down.